docker-compose.test.yml

```diff
@@ -1,3 +1,3 @@
 sut:
-  build:
-  command: ./run_tests.sh
\ No newline at end of file
+  build: .
+  command: ./run-test.sh
```

Comment:
Looks like your run-test.sh file is not marked as an executable file.
Try changing this line from

```bash
  command: ./run-test.sh
```

to

```bash
  command: bash ./run-test.sh
```